File: app/src/main/java/org/michaelbel/tjgram/data/entity/AttachResponse.java
package org.michaelbel.tjgram.data.entity;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

public class AttachResponse implements Serializable {
    @Expose @SerializedName("type") public String type;
    @Expose @SerializedName("data") public Attach data;
}

File: app/src/main/java/org/michaelbel/tjgram/data/entity/AttachResponse.kt
package org.michaelbel.tjgram.data.entity

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

import java.io.Serializable

data class AttachResponse (
    @Expose @SerializedName("type") var type: String,
    @Expose @SerializedName("data") var data: Attach
): Serializable